model/db/heart_disease_bycounty_ddl.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE "State_symbol" (
```

Comment:
Can you rename this to "States"

Reply:
-- State Name and Abbreviation
CREATE TABLE States (
    State TEXT (50),
    Abbr  TEXT (2) 
);